xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -1931,21 +1931,23 @@ void COutput::InitCycle()
 
     if (m_pp && !m_pp->UpdateDeintMethod(method))
     {
+      CLog::Log(LOGDEBUG, LOGVIDEO, "VAAPI output: Current postproc does not want new deinterlace mode, removing");
       delete m_pp;
       m_pp = NULL;
       DropVppProcessedPictures();
-      m_config.processInfo->SetVideoDeintMethod("unknown");
```

Comment:
why do you drop this?

Reply:
It will be overwritten by either the correct implementation or `skip` as fallback anyway (which I added), so it is not needed anymore.

Reply:
it reflects the state at this time. if anything goes wrong later, it won't get overwritten
keep it